news/9498.feature.rst

```diff
@@ -0,0 +1 @@
+Cache http responses from trusted hosts.
```

Comment:
I’d feel this is a bit vague if I read it in the release notes. Perhaps something like *If a host is explicitly specified as trusted (via the `--trusted-host` option), cache HTTP responses from it in addition to HTTPS ones.* would be clearer?

Reply:
nice idea, done!